packages/tooling/node/src/config.ts

```
@@ -105,6 +107,13 @@ const withDefault = (
       networkName,
       rpcUrlOverride ?? networkConfig?.url
     )
+    const buyerAccountOverrides = {
+      accountAddress: process.env.SUI_BUYER_ACCOUNT_ADDRESS,
+      accountPrivateKey: process.env.SUI_BUYER_ACCOUNT_PRIVATE_KEY,
+      accountMnemonic: process.env.SUI_BUYER_ACCOUNT_MNEMONIC
+    }
+    const hasBuyerAccountConfig = Object.values(buyerAccountOverrides).some(Boolean)
```

ESLint

[error] Replace `Boolean` with `⏎······Boolean⏎····`

(prettier/prettier)